blaze-server/src/test/scala/org/http4s/blaze/server/BlazeServerSuite.scala

```diff
@@ -63,8 +63,6 @@ class BlazeServerSuite extends Http4sSuite {
     )
   }
 
-  override def afterAll(): Unit = ioRuntime.shutdown()
-
```

Comment:
Ummm????

Reply:
Aha, `BlazeServerSuite` has its own runtime that needs to be shutdown. But it got renamed, and this was shutting down the global.